### DISCUSSION TITLE
Disable Model Saving and Chart Visualization

### DISCUSSION THREAD

**Discussion by zuliani99** [autogluon/autogluon#1062] [Q&A] [answered]
Hi,
Is it possible to disable the creation of the directory where models and intermediate outputs will be saved?
In addition to that is it possible to disable the visualization of the chart after the prediction result?
Thanks a lot

**Comment by Innixma** [comment]
Could you provide a code example that results in a chart being visualized? I don't believe that should be happening. Also make sure you are using the latest release (v0.1.0).

If you are referring to Tabular, it is not easy to avoid the creation of intermediate outputs and models to disk, as otherwise all models would need to be persisted in memory, which could cause out-of-memory errors. We would only consider this if there were strong reasons why the instance can't rely on disk for model persistence.

**Comment by zuliani99** [comment]
I mean after the end of the fit() summary autogluon will display in my browser all the model produced during fit(), is it possible to disable this features?

**Comment by bluelu** [comment]
+1 for this feature. I execute autogluon through a java process and it hangs trying to open up a browser at the end. Certainly I could send some interrupt, but this makes it more complex.

I uninstalled bokeh-2.0.1 for now, which breaks the generation of the report, and thus also does not trigger the browser open event.

**Comment by Innixma** [comment, marked as answer]
This has been resolved via #1166
